plugins/modules/helm_plugin.py

```diff
@@ -198,6 +198,8 @@ def main():
                     continue
                 name, dummy, dummy = line.split('\t', 3)
                 name = name.strip()
+                if name == "":
```

Comment:
what if we put this piece of code which is common to the 3 modules into a function in module_utils/helm.py ?

Reply:
Yes, we can. Thanks,